### ISSUE TITLE
The map is not showing the marker (odoo 15.0)

### ISSUE THREAD

**Issue by selvamani-ramasamy** [gityopie/odoo-addons#196]
I have installed the web_google_maps and the map view is coming up.  But it is not showing any marker for the locations.  I have updated the lang, lat for the contacts.  But still it does not work.

![Screen Shot 2022-07-09 at 4 08 32 AM](https://user-images.githubusercontent.com/17324206/178103143-ce7c95dc-47cb-4659-ab18-dc631356150d.png)
.

**Comment by selvamani-ramasamy** [comment]
When I click on the contacts in the side bar, it focuses to that location, but it does not show the marker.

**Comment by selvamani-ramasamy** [comment]
@gityopie Could you please help on this?

**Comment by selvamani-ramasamy** [comment]
Update:  it is working for res.partner module, but it does not work for  the module which is inherited from res.partner.

**Comment by gityopie** [comment]
@selvamani-ramasamy did you mean new model inherit `res.partner`?
the `google_map` view is not automatically created, it must define it manually

**Comment by selvamani-ramasamy** [comment]
@gityopie yes, the new module is inherited from res.partner.  I have modified the view to use the new module instead of res.partner.  The attached snapshot is for new module.  It loads the contacts on the right side bar.  When we click, then it will focus to that coordinates in the map.  But it is not marking the contact information in the map.

**Comment by gityopie** [comment]
@selvamani-ramasamy looks like a marker issue. Have you check on the network logs?

**Comment by selvamani-ramasamy** [comment]
@gityopie I have enabled all dev logs (--dev=qweb,xml,werkzeug) and I don't see any error or warning in the log console. Am I missing something?  Let me know how to check network logs specifically.

**Comment by gityopie** [comment]
@selvamani-ramasamy The network logs on the browser
